docs/source/contents/4_draft_mapping.rst

```diff
@@ -1331,6 +1331,297 @@ location
+    <recordInfo>
+        <recordIdentifier>record_Pseudolarix_0858</recordIdentifier>
+    </recordInfo>
+
+languageOfCataloging
+--------------------
+
+Use Case
+^^^^^^^^
+
+All of the recently migrated SCOUT to TEI collections (e.g. American Civil War Collection, Tennessee Documentary History, etc.)
+as well as some of UTK's less recent collections (e.g. Sanborn, mpabaker, etc.) contain the element languageOfCataloging.
+In total, it is found in approximately 6,000 records. Note that in all cases the language is English, but this information
+is represented as both a code ('eng') and a text value ('English').
+
+Justification
+^^^^^^^^^^^^^
+
+Currently <languageOfCataloging> is not publicly displayed anywhere outside of the MODS XML. The values of this element
+do have the potential to be used if UTK has materials that might warrant cataloging in another language, but currently
+this is not the case. An example of a project that includes two records, one catalogued in Spanish and one in English, is
+`UNC's New Roots / Nuevas Raíces <https://newroots.lib.unc.edu/>`_. While UTK may want to pursue a project like this in the
+future, presently it seems unlikely that it will. More importantly, if such a project became a priority, it would not be
+difficult to distinguish via code UTK's existing English records from records in another language. If we did want to create
+a project like this, information on the language of cataloging could be added across the repository with minimal effort.
+
+Xpath
+^^^^^
+
+"mods:recordInfo" / "mods:languageOfCataloging"
+
+Decision
+^^^^^^^^
+
+Since we are not currently utilizing these values in any way, these should be dropped in the mapping.
+
+Here's an `example record - sanborn:1002 <https://digital.lib.utk.edu/collections/islandora/object/sanborn%3A1002/datastream/MODS/view>`_.
+
+.. code-block:: xml
+
+    <recordInfo>
+        <languageOfCataloging>
+            <languageTerm authority="iso639-2b" type="code">eng</languageTerm>
+        </languageOfCataloging>
+    </recordInfo>
+
+recordOrigin
+------------
+
+Use Case
+^^^^^^^^
+
+The <recordOrigin> element includes information about what methods or transformations were used to prepare a record. There
+are six different distinct values in UTK's metadata.
+
+Justification
+^^^^^^^^^^^^^
+
+Because the existing values all relate to MODS XML, the string values present will no longer be applicable in a RDF-based
+platform. Discussion indicated that there might be some use for the general property of recordOrigin if a link to this
+mapping document or some other relevant resource was shared in place of the existing values. This administrative information
+could also be shared on the Digital Collections website or elsewhere rather than the record. As a convincing argument
+was not made that this information is essential, it was decided to drop these values
+
+Xpath
+^^^^^
+
+"mods:recordInfo" / "mods:recordOrigin"
```

Comment:
Normally, we express an xpath here.  Are you trying to say:

```
mods:recordInfo/mods:recordOrigin
```

Reply:
This was resolved in https://github.com/UTKcataloging/mods_to_rdf/pull/27/commits/2bd47b57b297b1805e1b9a7e6f517ce517eda056